src/DepOutputChannel.ts

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class DepOutputChannel {
+  outputChannel: vscode.OutputChannel;
+  constructor(channelName = 'Dependency Analytics') {
+    if (!this.outputChannel) {
+      this.outputChannel = vscode.window.createOutputChannel(channelName);
+    }
+  }
+
+  getOutputChannel(): vscode.OutputChannel {
+    this.outputChannel.clear();
+    return this.outputChannel;
+  }
+
+  showOutputChannel() {
```

Comment:
and provide type for the function

Reply:
You mean return type i.e `void` in this scenario?

Reply:
yes